lib/io/stream/writable-object.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Interface for core functionality of write streams.
```

Comment:
I believe the comment should not be necessary here as the interface is small and the purpose should be clear from the name and function names. Instead, if you want to include the comment, perhaps explain whose responsibility is to call the close method.